bin/node/runtime/src/lib.rs

```diff
@@ -513,6 +513,7 @@ parameter_types! {
 		.get(DispatchClass::Normal)
 		.max_extrinsic.expect("Normal extrinsics have a weight limit configured; qed")
 		.saturating_sub(BlockExecutionWeight::get());
+	pub MinerMaxLength: u32 = 2 * 1024 * 1024; // 2Mb
```

Comment:
it should be computed from BlockLength type. something like

`RuntimeBlockLength::get(DispatchClass::Normal)`

But I am not sure the result is actually accurate and if we should substract some BlockBaseLength 🤔

Reply:
Yes, that's what [the real solutions do](https://github.com/paritytech/polkadot/pull/2649/files#diff-3287a4ee060b1070a8e6559a094b4890a596a1b281effef54cafe59da20edf49R126-R129).

2Mb is just Some Value for `bin/node/runtime`, which I've been informed is not a serious runtime.

Reply:
I think it is better to have it implemented with best practice anyway, copying what is in polkadot would be better IMHO

Reply:
Ok, c230d911c.